cli/command_policy_set_retention.go

```diff
@@ -38,6 +40,7 @@ func (c *policyRetentionFlags) setRetentionPolicyFromFlags(ctx context.Context,
 		{"number of daily backups to keep", &rp.KeepDaily, c.policySetKeepDaily},
 		{"number of hourly backups to keep", &rp.KeepHourly, c.policySetKeepHourly},
 		{"number of latest backups to keep", &rp.KeepLatest, c.policySetKeepLatest},
+		{"minimun number of days to keep", &rp.KeepMinDays, c.policySetKeepMinDays},
```

Comment:
please fix the typo "minimun", also rename the fields to match the flags.

the description of the flag is too similar to "number of daily backups to keep". 
How about "minimum number of days to retain the snapshot after creation" or something to that effect